src/test/runner.ts

```diff
@@ -169,6 +170,14 @@ export class Runner {
     let globalSetupResult: any;
     if (config.globalSetup)
       globalSetupResult = await this._loader.loadGlobalHook(config.globalSetup, 'globalSetup')(this._loader.fullConfig());
+    let webServer: WebServer|null = null;
+    try {
+      if (config.webServer)
+        webServer = await WebServer.create(config.webServer);
+    } catch (error) {
+      console.log(error instanceof Error ? error.message : error);
```

Comment:
Let's throw instead? We throw in many places here and get default reporting of the error.